server/package.json

```diff
@@ -5,10 +5,7 @@
   "main": "server.js",
   "dependencies": {
     "body-parser": "^1.18.3",
-    "chai": "^4.2.0",
-    "chai-http": "^4.2.0",
-    "chalk": "^2.4.2",
-    "connect-nedb-session": "0.0.3",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-odetect-fixes",
```

Comment:
This will need to be updating to the correct version after this PR is merged: bravetechnologycoop/brave-alert-lib#3

Reply:
Done.